ledger/src/blockstore_db.rs

```diff
@@ -1483,20 +1501,19 @@ where
     /// Create a key type suitable for use with multi_get_bytes() and
     /// multi_get(). Those functions return iterators, so the keys must be
     /// created with a separate function in order to live long enough
-    pub(crate) fn multi_get_keys<I>(&self, keys: I) -> Vec<Vec<u8>>
+    pub(crate) fn multi_get_keys<I>(&self, keys: I) -> Vec<<C as Column>::Key>
```

Comment:
Refactoring aside, getting all of the keys in one contiguous `Vec` instead of a vector of vectors is a nice win. I had this as a low priority item on TODO list but your solution does it in a cleaner way than what I had in mind.

Reply:
> one contiguous `Vec`(`Vec<[u8; _]>`)

I am guessing the 2nd `Vec` was a copy-paste mistake here, but just to be clear, the return type here is just 

    ->  Vec<[u8; _]>

(i.e. there is no 2nd nested `Vec`).

Reply:
> (i.e. there is no 2nd nested Vec).

Yep, confusing wording on my side; updated for the sake of any future readers